pkg/app/pipectl/cmd/application/add.go

```diff
@@ -0,0 +1,108 @@
+// Copyright 2020 The PipeCD Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package application
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/pipe-cd/pipe/pkg/app/api/service/apiservice"
+	"github.com/pipe-cd/pipe/pkg/cli"
+	"github.com/pipe-cd/pipe/pkg/model"
+)
+
+type add struct {
+	root *command
+
+	appName       string
+	appKind       string
+	envID         string
+	pipedID       string
+	cloudProvider string
+
+	gitPathRepoID         string
+	gitPathAppDir         string
+	gitPathConfigFileName string
+}
+
+func newAddCommand(root *command) *cobra.Command {
+	c := &add{
+		root:                  root,
+		gitPathConfigFileName: ".pipe.yaml",
```

Comment:
I feel like it should be changeable via `--config-file-name` or something like that. And the default value should refer to `pkg.model.DefaultDeploymentConfigFileName`.

Reply:
Sorry, I just now found it has been already changeable.

Reply:
👍  I changed to use the value defined in the model package and simplified the flags.